### PR TITLE
Adding envar to databox and prod - readmechanges

### PR DESCRIPTION
**Pull request by ghasani**
Adding instructions on how to add new envar to envars.yml on a databox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +83,20 @@ MY_SPECIAL_VAR:
...
+1. Navigate to the beetle folder
+2. Run ```envars add VAR_NAME=value -s```
+3. One the above line is ran, the terminal will print the secret to the variable like below:
+
+```
+default: !secret |-
+      AQICAHiOrkzo4U0Qe3l3gBab8dyFfpnlamupaQ8EuHEULsA6uwGWjI8fnd7xt
+```
+
+4. Copy and paste this into the envars.yml file and include it in your PR.
```
Comment (kthhrv) [inline README.md:101]: the `envars` command add the new var to the file automatically, no copy and paste required

File: README.md [hunk 2]
```diff
@@ -83,3 +83,20 @@ MY_SPECIAL_VAR:
...
+5. For the new variable to be added in your databox, restart your containers.
```
Comment (kthhrv) [inline README.md:102]:
```suggestion
5. For the new variable to be added in your databox, restart your containers.
```
beetle $ invoke restart
```
```

**Review by kthhrv — CHANGES_REQUESTED** [review]
nearly ;-)

**Comment by ghasani** [comment]
@kthhrv updated

File: README.md [hunk 3]
```diff
@@ -91,12 +91,6 @@ You may want to add a new enviroment variable to your databox for testing and th
...
-
-```
-default: !secret |-
-      AQICAHiOrkzo4U0Qe3l3gBab8dyFfpnlamupaQ8EuHEULsA6uwGWjI8fnd7xt
-```
-
-4. Copy and paste this into the envars.yml file and include it in your PR.
-5. For the new variable to be added in your databox, restart your containers.
\ No newline at end of file
+3. One the above line is ran, the new enviroment variable will be added to the envars.yml file.
```
Comment (kthhrv) [inline README.md:94]:
```suggestion
3. Once the above command is run, the new environment variable will be added to the envars.yml file.
```

**Review by kthhrv — CHANGES_REQUESTED** [review]
;-)

**Comment by kthhrv** [comment]
this shouldn't be in the envars repo, it should be in the beetle one ;-)

**Pull request closed without merging**